test/unit/src/common/components/user-avatar/t_user-avatar.js

```diff
@@ -1,22 +1,41 @@
 import React from 'react';
 import expect from 'expect';
 import { shallow } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
 
-import { UserAvatar } from '../../../../../../src/common/components/user-avatar';
+import UserAvatar from '../../../../../../src/common/components/user-avatar';
 import { HeadingOne, HeadingThree } from '../../../../../../src/common/components/vanilla/heading';
 
+const middlewares = [ thunk ];
+const mockStore = configureMockStore(middlewares);
+
 describe('<UserAvatar />', function() {
   let element;
+  let store;
+  let auth;
+  let user;
+
+  beforeEach(() => {
+    store = mockStore({
+      auth,
+      user
+    });
+
+    // shallow render container component with the store
+    element = shallow(<UserAvatar store={store} />)
```

Comment:
@katiefenn @earnubs 

Would it be better if we start call presentational component 'Views'? Like:

```js

export class UserAvatarView extends Component {
}

export default connect(...)(UserAvatarView);
```

This would be less confusing when importing.
Of course this leaves the question how do we call purely presentational components that don't have `connect` at all, like vanilla ones, etc.